### ISSUE TITLE
Ubuntu release unsupported

### ISSUE THREAD

**Issue by paulmenzel** [grml/grml-debootstrap#180]
debootstrap also supports Ubuntu releases. Using `--release bionic` results in an error. For example, the script `chroot-script` adds an entry for the security package archive.

https://github.com/grml/grml-debootstrap/blob/59e75958079096421a679c83e2526c973038ba83/chroot-script#L113-L127

**Comment by mika** [comment]
Yeah, but grml-debootstrap is only about Debian so far, as the package description (`ease installation of a pure Debian system`) says, as well as the manual page at the `SUPPORTED RELEASES` section. :)

Furthermore quoting from `TODO`:

* test support for Ubuntu as debootstrap >=1.0.0 provides support for it

I'm not objecting to also support Ubuntu (and therefore adjusted the title of this issue), but I don't have a use case on my own, so patches welcome. :)
